src/components/sections/favorite-departure-item.tsx

```diff
@@ -6,10 +6,10 @@ import {
   TouchableOpacity,
   View,
 } from 'react-native';
-import SvgDelete from '../../assets/svg/icons/actions/Delete';
-import {StoredFavoriteDeparture} from '../../favorites/types';
-import {useTheme} from '../../theme';
-import {SectionTexts, useTranslation} from '../../translations';
+import SvgDelete from '@atb/assets/svg/icons/actions/Delete';
+import {StoredFavoriteDeparture} from '@atb/favorites/types';
+import {useTheme} from '@atb/theme';
+import {SectionTexts, useTranslation} from '@atb/translations';
 import {screenReaderPause} from '../accessible-text';
 import ThemeText from '../text';
```

Comment:
The logic here is that if the import is just one level down then `@atb/` is not used?

Reply:
down = up

Reply:
Not explicitly there might be somewhere it's missing (especially with `../` as I didn't manually go through every file). But the logic should be:

- If it is part of module/component: use relative
- If it imports something from another module/component: use absolute.

Reply:
FWIW, I'm updating for `components/` now